recipe_scrapers/__init__.py

```diff
@@ -438,7 +438,7 @@
 }
 
 
-def scrape_me(url_path, **options):
+def scrape_me(url_path: str, **options: Any) -> AbstractScraper:
```

Comment:
Shouldn't `options` be `Dict[string]Any`

Reply:
Oh and `Optional`

Reply:
updated, I think it should just be `dict[str, Any]` not `dict[str, Any] | None` since it's using the `**` thing